android-exoplayer/build.gradle

```diff
@@ -63,10 +61,6 @@ dependencies {
 
     implementation 'androidx.constraintlayout:constraintlayout:1.1.3'
 
-    implementation(project(':dice-shield-android')) {
```

Comment:
`doris-android` brings the `dice-shield-android`, no need to resolve it through npm